capirca/aclgen.py

```diff
@@ -183,6 +183,7 @@ def RenderFile(base_directory: str, input_file: pathlib.Path,
   msmpc = False
   spd = False
   nsx = False
+  nsxt1 = False
```

Comment:
Why `nsxt1` instead of `nsxt`? 

Is this because the module is called `nsxt` and you don't want to overwrite the global variable referencing the `nsxt.py` module? Can we call this something different, e.g. `nsxt_pol`?

Reply:
Yes, having this as nsxt was overwriting the global and code didn't work. I could rename it to nsxt_pol, my mind just blanked at the time for a good variable name.